doc/build-freebsd.md

```diff
@@ -47,13 +112,20 @@ With wallet:
     MAKE=gmake
```

Comment:
I never needed to append `MAKE=gmake` to `./configure`. Just running `gmake` later suffices. If that works for you too, then I would suggest to remove it from here.

Reply:
I don't seem to need `MAKE=gmake` either anymore—even on FreeBSD 12.1! It always crashed in the configure stage while resolving dependencies without that. Strange, must have changed with some autotools/autoconf update.

Reply:
Oh crap it's back. This seems to happen not initially, but after `make` automatically decides to re-run configuref
```
config.status: error: in `/home/user/src/bitcoin':
config.status: error: Something went wrong bootstrapping makefile fragments
    for automatic dependency tracking.  If GNU make was not used, consider
    re-running the configure script with MAKE="gmake" (or whatever is
    necessary).  You can also try re-running configure with the
    '--disable-dependency-tracking' option to at least be able to build
    the package (albeit without support for automatic dependency tracking).
See `config.log' for more details
gmake: *** [Makefile:718: Makefile] Error 1
```

Reply:
Ah, than I guess `MAKE=gmake` should stay.

Thanks for the hint!